airflow/airflow.go

```diff
@@ -58,7 +58,7 @@ func initFiles(root string, files map[string]string) error {
 }
 
 // Init will scaffold out a new airflow project
-func Init(path string, airflowImageTag string) error {
+func Init(path, airflowImageTag string) error {
```

Comment:
Did the linter remove `string` here?

Reply:
@adam2k no, it's another style to group variables with same type:
```
path string, airflowImageTag string
or
path, airflowImageTag string
```